addons/flannel/template/base/install.sh

```diff
@@ -114,6 +116,60 @@ function weave_to_flannel() {
 
     logStep "Applying Flannel"
     kubectl -n kube-flannel apply -k "$dst/"
+
+    # if there is more than one node, prompt to run on each primary/master node, and then on each worker/secondary node
+    local master_node_count=
+    master_node_count=$(kubectl get nodes --no-headers --selector='node-role.kubernetes.io/control-plane' | wc -l)
+    local hostnamevar
+    hostnamevar=$(hostname)
+    local master_node_names=
+    master_node_names=$(kubectl get nodes --no-headers --selector='node-role.kubernetes.io/control-plane' -o custom-columns=NAME:.metadata.name | grep -v "$hostnamevar")
+    if [ "$master_node_count" -gt 0 ]; then
+        printf "${YELLOW}Moving primary nodes from Weave to Flannel requires removing certain weave files and restarting kubelet.${NC}\n"
+        printf "${YELLOW}Please run the following command on each of the listed primary nodes:${NC}\n\n"
```

Comment:
Could we not run the commands instead of ask for the user to do it?

Reply:
We don't have a way to run commands on other nodes ourselves